tests/net/lib/coap/src/main.c

```diff
@@ -1490,6 +1490,68 @@ static int test_parse_malformed_opt_len_ext(void)
 	return result;
 }
 
+/* IPv6 + UDP frame (48 bytes) */
+static const unsigned char ipv6_malformed_marker[] = {
+	/* IPv6 header starts here */
+	0x60, 0x00, 0x00, 0x00, 0x00, 0x0D, 0x11, 0xFF,
```

Comment:
0x60, 0x00, 0x00, 0x00, 0x00, ""0x0E"", 0x11, 0xFF,

Reply:
You are right. I forget to update the payload length of both ipv6 and udp header. Thanks for remind